x-pack/plugins/security_solution/public/management/pages/event_filters/view/event_filters_list_page.tsx

```diff
@@ -103,6 +105,13 @@ export const EventFiltersListPage = memo(() => {
     }
   }, [dispatch, formEntry, history, isActionError, location, navigateCallback]);
 
+  const backButton = useMemo(() => {
+    if (routeState && routeState.onBackButtonNavigateTo) {
+      return <BackToExternalAppButton {...routeState} />;
+    }
+    return null;
```

Comment:
nitpick: does [headerComponent](https://github.com/elastic/kibana/pull/101280/files#diff-7ebca7d3befeba3652d93b7ef771963b64006f2fccb16812cf943752c2a153b8R185) expect a `null` instead of `undefined`?

Reply:
Yes, it's an optional component, so null works in this case